modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceSAPConstants.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.constants;
+
+import com.liferay.commerce.account.service.CommerceAccountService;
+import com.liferay.commerce.service.CommerceCountryService;
+import com.liferay.commerce.service.CommerceOrderItemService;
+import com.liferay.commerce.service.CommerceOrderService;
+import com.liferay.commerce.service.CommerceRegionService;
+import com.liferay.petra.string.StringBundler;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceSAPConstants {
+
+	public static final String CLASS_NAME_COMMERCE_CART_RESOURCE =
+		"com.liferay.commerce.frontend.internal.cart.CommerceCartResource";
+
+	public static final String CLASS_NAME_COMMERCE_HEADLESS_CART_ITEM_RESOURCE =
+		"com.liferay.headless.commerce.delivery.cart.internal.resource.v1_0." +
+			"CartItemResourceImpl";
+
+	public static final String CLASS_NAME_COMMERCE_HEADLESS_CART_RESOURCE =
+		"com.liferay.headless.commerce.delivery.cart.internal.resource.v1_0." +
+			"CartResourceImpl";
+
+	public static final String CLASS_NAME_COMMERCE_SEARCH_RESOURCE =
+		"com.liferay.commerce.frontend.internal.search.CommerceSearchResource";
+
+	public static final String COMMERCE_SAP_ENTRY_NAME = "COMMERCE_DEFAULT";
```

Comment:
@luca-pellizzon I think we should just call this 
```suggestion
	public static final String _SAP_ENTRY_NAME = "COMMERCE_DEFAULT";
```